addons/microsoft_calendar/models/calendar.py

```diff
@@ -104,7 +104,7 @@ def _check_organizer_validation(self, sender_user, partner_included):
                 raise ValidationError(
                     _("For having a different organizer in your event, it is necessary that "
                       "the organizer have its Odoo Calendar synced with Outlook Calendar."))
-            elif not partner_included:
+            elif sender_sync_status and not partner_included:
```

Comment:
`_check_microsoft_sync_status` is doing that:

```py
def _check_microsoft_sync_status(self):
        """
        Returns True if synchronization with Outlook Calendar is active and False otherwise.
        The 'microsoft_synchronization_stopped' variable needs to be 'False' and Outlook account must be connected.
        """
        outlook_connected = self.env.user._get_microsoft_calendar_token() and self.env.user._is_microsoft_calendar_valid()
        return outlook_connected and self.env.user.microsoft_synchronization_stopped is False
```
and `_get_microsoft_calendar_token` is doing this:
```py
def _get_microsoft_calendar_token(self):
        if not self:
            return None

        self.ensure_one()
        if self.microsoft_calendar_rtoken and not self._is_microsoft_calendar_valid():
            self._refresh_microsoft_calendar_token()
        return self.microsoft_calendar_token
```
`_is_microsoft_calendar_valid` is called twice. If the token is no longer valid, it will be refreshed. Why are you checking that it is valid in `_check_microsoft_sync_status` if `_get_microsoft_calendar_token` is able to do it ?